DataFormats/Detectors/ITSMFT/common/src/TopologyDictionary.cxx

```diff
@@ -83,7 +83,7 @@ ClassImp(o2::ITSMFT::TopologyDictionary)
         in.read(reinterpret_cast<char*>(&gr.mPattern.mBitmap), sizeof(unsigned char) * (ClusterPattern::kExtendedPatternBytes));
         mVectorOfGroupIDs.push_back(gr);
         if (gr.mPattern.getUsedBytes() == 1)
-          mSmallTopologiesLUT[(gr.mPattern.getRowSpan() - 1) * 255 + (int)gr.mPattern.mBitmap[2]] = groupID;
+          mSmallTopologiesLUT[(gr.mPattern.getColumnSpan() - 1) * 255 + (int)gr.mPattern.mBitmap[2]] = groupID;
```

Comment:
is this a bugfix? Or a change in the access sequence?

Reply:
It is a bug fix. I changed from Rows to Columns everywhere mSmallLUT was called, because otherwise the access to a specific element was not unique for a single topology.